test/integration/server.js

```diff
@@ -153,10 +153,12 @@ helpers.stubUtxos = function(server, wallet, amounts, cb) {
       return {
         txid: helpers.randomTXID(),
         vout: Math.floor(Math.random() * 10 + 1),
-        satoshis: helpers.toSatoshi(amount).toString(),
+        satoshis: helpers.toSatoshi(amount),
```

Comment:
You are right. I remember having to deal with a string but maybe that was when BWS was using bitcore-explorers to get UTXOs list. Anyways, good catch!